certified-connectors/QPP NG Integration Hub/ReadMe.md

```diff
@@ -64,179 +63,342 @@ Copy and paste the Below json formatted using online json compiler to the Body p
-
-| Parameter            | Defintion                                                                           | Example                                                                                                                                                                                                                                             |
-| -------------------- | ----------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| new_url                 | Url for the Host environment that This Action needs to points to.                   | {HostNameHere}.app.quark.com                                                                                                                                                                                                                        |
-| assetId              | Asset Id which needs to be checked Out                                              | 50                                                                                                                                                                                                                                                  |
-| AttributeValue Array | Paste the example exactly into the body of the action , after switching to raw view | [ {<br>   "attributeId": 16,<br>   "type": 1,<br>   "attributeValue": {<br>   "value": "c:/checkedoutfiles"<br>  }<br> },<br>  {<br>   "attributeId": 34,<br>   "type": 1,<br>   "attributeValue": {<br>   "value": "REST-CLIENT"<br>  }<br> }<br>] |
+This operation downloads a file or Blob from the system using the specified context and file identifier. This action is mostly used in conjunction with _getAssetDownloadInfo_ and _publishAsset_ actions as they return the blob location and other attributes in response. With the help of this action, you can download the asset from the blob. Just like Upload, it’s a 2-step process. Please be careful as it’s time bound and will expire in 120 secs. 
+
+More information can be found on the [Quark Developer portal](https://qppngsdk-rel3x.app.quark.com/Documentation/platformServices/restapi/api.html#Quark_application_API_3_0_0_SNAPSHOT_download).
+
+To use this action, provide the following param's – 
+ - **Auth Token for Header:** This action is depenedant on the **Get Token as per Grant Type** action to get authentication token for this param. Considering this action is added before Download (Blob) action and you have kept its name as Get **Token as per Grant Type**, then add this value in Expression editor next to dynamic content: _outputs('Get_Token_as_per_Grant_Type')?['body/access_token']_. For more information, refer to **Get Token as per Grant Type** section.
```

Comment:
please correct the spelling "depenedant"

Reply:
Done